src/components/utils/checkbox-list.jsx

```diff
@@ -5,74 +5,41 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import React, { useCallback, useEffect, useRef, useState } from 'react';
-import List from '@mui/material/List';
+import { useRef } from 'react';
 import PropTypes from 'prop-types';
+import { List } from '@mui/material';
+
+function defaultComparator(a, b) {
+    return a.id === b.id;
+}
 
 const CheckboxList = ({
     itemRenderer,
-    toggleSelectAll,
     values,
     onChecked,
-    initialSelection,
-    itemComparator = (a, b) => a.id === b.id,
+    checkedValues,
+    itemComparator = defaultComparator,
     ...props
 }) => {
-    const [checked, setChecked] = useState(new Set(initialSelection));
+    const valuesRef = useRef();
```

Comment:
i don't see where this variable are used !?

Reply:
arf forget to clean this